src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Circle extends AbstractFigure {
+    private final Random random = new Random();
```

Comment:
Create the appropriate variables
Random should not be a variable of shapes

Rondom values must be written using a constructor
```suggestion
```